ansible_base/authentication/authenticator_plugins/base.py

```diff
@@ -134,3 +135,56 @@ def add_related_fields(self, request, authenticator):
 
     def validate(self, serializer, data):
         return data
+
+    def move_authenticator_user_to(self, new_user, old_authenticator_user):
+        """
+        new_user: django User instance. User that we're moving this account to.
+        old_authenticator_user: AuthenticatorUser instance from this authenticator that is being removed.
+        """
+        exclude_fields = (
+            "social_auth",
+            "authenticator_users",
+            "groups",
+            "has_roles",
+            "role_assignments",
+        )
+
+        old_user = old_authenticator_user.user
+
+        # Delete the old authenticator user
+        old_authenticator_user.delete()
+
+        if new_user.pk == old_user.pk:
+            return
```

Comment:
It doesn't have to create the new `AuthenticatorUser`? That's happen through normal auth logic, and by deleting the `old_authenticator_user`, that is unblocked?